install_config/configuring_vsphere.adoc

```diff
@@ -157,6 +157,27 @@ determined by using the command `uname -n`. In case of a mismatch, the native
 cloud integration for those providers will not work.
 ====
 
+== Configuring OpenShift Container Platform for vSphere using Ansible
+
+You can configure OpenShift Container Platform for VMware vSphere (VCP) by modifying the Ansible inventory file at installation time or after installation.
+
+.Procedure
+
+. Add the following to the Ansible inventory file:
```

Comment:
s/following/following section

Reply:
I've updated and squashed the commits, so the review displayed as `Outdated`.

Reply:
This looks pretty good @bysnupy. Thank you. :)

The only other change I'd make is to use {product-title} in line 162, too.

I hope QE approves the change.